Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -132,7 +154,7 @@ private static void LoadLocalizationFile(string path, string localizationCode)
         /// <summary>
         /// Returns the localization for the given key, or the key itself, if no translation exists.
         /// </summary>
-        private static string GetLocalization(string key, FallbackMode fallbackMode, string language, params string[] additionalValues)
+        public static string GetLocalization(string key, FallbackMode fallbackMode, string language, params string[] additionalValues)
```

Comment:
this hsould remain private. a public version exists near line 60 `public static string GetLocalization(string key, params string[] additionalValues)` which encapsulates this version